conf/evolutions/084-zarr-private-links.sql

```diff
@@ -0,0 +1,24 @@
+-- Note that this evolution introduces constraints which may not be met by existing data. See migration guide for manual steps
+
+START TRANSACTION;
+
+CREATE TABLE webknossos.annotation_private_links
```

Comment:
I guess it should be like this?
```suggestion
CREATE TABLE webknossos.annotation_privateLinks
```

Reply:
Hm. I don't know. The `annotation_privateLinks` version leads to weird Table names in the Scala code (`AnnotationPrivatelinks` instead of `AnnotationPrivateLinks`)

Reply:
What about `CREATE TABLE webknossos."annotation_privateLinks"`?

Reply:
I’m also in favor of `annotation_privateLinks` (like norman said), that is our convention. The generated slick classes do have weird names, but they are only accessed in very few places, so I’d say it’s fine